### DISCUSSION TITLE
dashboard export permissions

### DISCUSSION THREAD

**Discussion by lucas7617** [grafana/grafana#71566] [General]
Being able to manage viewer user permissions so they can't export the dashboard or panel, this can only be done by admins to prevent others from downloading the dashboard we made.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82609. Feel free to continue the discussion around this there. Thank you!
